README.md

```diff
@@ -1,4 +1,7 @@
-<p align="center"><img src="/art/readme.png" alt="Logo Laravel Forge CLI" style="width:70%;"></p>
+<p align="center">
+    <img src="/art/header.png" alt="Laravel Forge CLI header with logo">
+    <img src="/art/readme.png" alt="Logo Laravel Forge CLI" style="width:70%;">
```

Comment:
Two images here seem like a bit much. Bit too much before the actual readme content is shown.

Reply:
What about using @nunomaduro's trick?

![image](https://user-images.githubusercontent.com/502041/127864141-a51e223d-46a7-4ac6-9fac-37c236e9fa52.png)

Reply:
Or, using the "header" in the header… and the preview in a "Preview" section along the README file.

Reply:
Both options sound find to me. Maybe check what @nunomaduro thinks.

Reply:
@taylorotwell decides, but personally I think is a good idea to have that logo on the right top of the console example we have, but a "colored" logo instead of gray.